main.cpp

```diff
@@ -327,27 +412,39 @@ struct Display
     std::string displayTechnology = "LED";
 
     
-    void displaySettings(char displayColor = 'r');
+    void displaySettings(char newDisplayColor);
     
     void energyConsumption(int wattsUsed);
     
     void displayBritenessDim (bool powerSaver = true, int lumens = 100);
 };
  
-void Display::displaySettings(char displayColor)
+void Display::displaySettings(char newDisplayColor)
 {
-
+    if (displayTechnology == "OLED")
+    {
+        newDisplayColor = 'r';
+    }
+    else
+    {
+        (newDisplayColor = 'b');
+    } 
+}
```

Comment:
You don't need parentheses around this statement in the `else` block
- [ ] revise